Generals/Code/GameEngine/Source/Common/RTS/ResourceGatheringManager.cpp

```diff
@@ -234,7 +234,8 @@ Object *ResourceGatheringManager::findBestSupplyCenter( Object *queryObject )
 
 		if( currentCenter == NULL )
 		{
-			iterator = m_supplyWarehouses.erase( iterator );
+			// TheSuperHackers @bugfix @ShizCalev 04/03/2025 Corrected wrong list being used - issue #342
```

Comment:
Please, remove these comments

Reply:
He added the comments because I have proposed it in the other Pull. I will be in favor of documenting every change we make, both in code and in a separate yaml file. We can debate whether we want to commit to that apprach.

In our data branch we do the exact same and it has worked well and there are many situations where a good comments helps when revisiting the code later.

Reply:
Git is just a tool and not bound to source code. But yes too many code comments will not make sense and clutter the code. Perhaps they do make sense above a function definition or at the top of a file. So that anyone looking at the code can read right there what was changed, without inquiring git history.

Personally, I think there is great value in seeing what someone has changed. Many programmers will not closely monitor the git history when reading code, but they will see in-code comments at wherever they are looking.